cdap-ui/app/cdap/components/SplashScreen/index.js

```diff
@@ -96,87 +85,70 @@ import T from 'i18n-react';
   toggleCheckbox() {
     this.doNotShowCheck = !this.doNotShowCheck;
   }
-  firstOnChange(e) {
-    this.setState({first : e.target.value});
-  }
-  lastOnChange(e) {
-    this.setState({last : e.target.value});
-  }
-  emailOnChange(e) {
-    this.setState({email : e.target.value});
-  }
   render() {
 
     let cardTitle = this.state.videoOpen ? '' : T.translate('features.SplashScreen.title');
+    let cardTitleTwo = this.state.videoOpen ? '' : T.translate('features.SplashScreen.titleTwo');
+
+    let cardHeader = (
+      <div className="card-header-splash">
+        <h3>{cardTitle}</h3>
+        <h3>{cardTitleTwo}</h3>
+        <div
+          className="fa fa-times"
+          onClick={this.onClose}
+        />
+      </div>
+    );
+
     return (
       <div className={!this.state.showSplashScreen ? 'hide' : ''}>
         <div className="splash-screen-backdrop"></div>
         <div className="splash-screen">
           <Card
             className="splash-screen-card"
-            closeable
-            title={cardTitle}
-            onClose={this.onClose.bind(this)}
+            header={cardHeader}
           >
             <div className="text-center">
-            <div className="splash-main-container">
-            {
-              this.state.videoOpen ?
-                <div className="cask-video-container">
-                  <CaskVideo />
-                </div>
-              :
-                <div>
-                  <span className="fa fa-5x icon-fist"></span>
-                  <div className="version-label">
-                    {T.translate('features.SplashScreen.version-label')}
+              <div className="splash-main-container">
+              {
+                this.state.videoOpen ?
+                  <div className="cask-video-container">
+                    <CaskVideo />
                   </div>
-                  <h4>
-                    {T.translate('features.SplashScreen.intro-message')}
-                  </h4>
-                </div>
-            }
-            </div>
-
+                :
+                  <div>
+                    <img width="100px" src="cdap_assets/img/cdaplogo_white.png" />
+                    <div className="version-label">
+                      {T.translate('features.SplashScreen.version-label')}
+                    </div>
+                    <h4>
+                      {T.translate('features.SplashScreen.intro-message')}
+                    </h4>
+                  </div>
+              }
+              </div>
               <br />
-              <div className={this.state.showRegistration ? 'group' : 'group no-registration'}>
-                <a className="spash-screen-btn" target="_blank" href="http://docs.cask.co/cdap">
+              <div className={'group'}>
+                <a className="spash-screen-btn" target="_blank" href="http://docs.cask.co/cdap/4.0.0-SNAPSHOT/en/index.html">
```

Comment:
Might be better to fetch this version dynamically from the backend, otherwise, this is another step in the release checklist